m2.md

```diff
@@ -0,0 +1,363 @@
+# Finding the Gaps: Mutation Testing for Move with move-mutation-tools
+
+In a [previous blog post](https://medium.com/aptoslabs/specification-testing-for-move-3ef8b5ced1f9), we explained the concept of mutation testing. Here we'll expand on
+that and explain new developments done to the tool.
+
+## Quick refresher: What mutation testing does
+
+Mutation testing deliberately introduces small, controlled bugs (mutants) into your Move code — changing an operator, constant, or condition — and re-runs your test suite.
+
+If a test fails, the mutant is killed. If all tests still pass, the mutant survives.
+
+Surviving mutants expose blind spots in your test suite — logic that could break without your tests noticing.
+
+Example:
+
+`assert!(signer::address_of(admin) == @decibel_dex, EINVALID_ADDRESS);`
+
+A mutant might flip the comparison:
+
+`assert!(signer::address_of(admin) != @decibel_dex, EINVALID_ADDRESS);`
+
+If your tests never call this function with the wrong signer, this mutant will survive — revealing that your test suite doesn’t verify access control properly.
+
+## New Features
+
+### Modes
+
+To optimize mutation testing by selecting operators based on their ability to detect test gaps, use the `--mode` option. Operators that produce more surviving mutants are more effective at revealing gaps in test coverage, as surviving mutants indicate untested code paths.
+
+```bash
+# Light mode - operators optimized for detecting test gaps with fewest mutants
+move-mutation-test run --output report.json --mode light
+
+# Medium mode - light + additional operators for broader test gap detection
+move-mutation-test run --output report.json --mode medium
+
+# Medium-only mode - only the operator added in medium (not including light)
+move-mutation-test run --output report.json --mode medium-only
+
+# Heavy mode - default, all operators for maximum test gap detection
+move-mutation-test run --output report.json --mode heavy
+
+# Heavy-only mode - only the operators added in heavy (not including light/medium)
+move-mutation-test run --output report.json --mode heavy-only
+```
+
+The modes include:
+
+- **light**: `binary_operator_swap`, `break_continue_replacement`, `delete_statement` (3 operators)
+- **medium**: light + `literal_replacement` (4 operators)
+- **medium-only**: `literal_replacement` (1 operator - only what's added in medium)
+- **heavy**: all 7 operators
+- **heavy-only**: `unary_operator_replacement`, `binary_operator_replacement`, `if_else_replacement` (3 operators - only what's added in heavy)
+
+The tool is quite resource-heavy and time-expensive, so the recommendation on how to use it most efficiently is:
```

Comment:
I think we need to give some additional context here (about why the tool is expensive to use, but yet is valuable).